ketcher-autotests/tests/test-data/bicycle.mol

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
put this file in corresponding MOL folder

Reply:
Done.